scripts/game.js

```diff
@@ -9,13 +12,40 @@ export default class Game {
         this.centerX = this.canvas.width / 2;
         this.centerY = this.canvas.height / 2;
         this.yinYang = new YinYang(this.centerX, this.centerY, state.playerRadius);
+        this.timer = new Timer();
         this.orbs = {};
         this.countOrbs = 0;
-        this.interval = setInterval(() => {
-            this.spawnOrb();
-        }, 500);
+        this.interval = null;
```

Comment:
NIT:  prob want a more descriptive name for this now that there are two intervals 🤔 
```suggestion
        this.orbSpawnInterval = null;
```

Reply:
will do in a later commit